shuup/front/locale/fi_FI/LC_MESSAGES/django.po

```diff
@@ -821,6 +821,27 @@ msgstr "Seuraava"
 msgid "Items %(start_index)d of %(end_index)d of %(count)d total"
 msgstr "Tuotteet %(start_index)d - %(end_index)d Yhteensä %(count)d"
 
+msgid "Change visibility limit to"
+msgstr "Vaihda näkyvyysrajoite"
+
+msgid "show only visible products and categories"
+msgstr "näytä vain näkyvät tuotteet ja kategoriat"
+
+msgid "show all products and categories"
+msgstr "näytä kaikki tuotteet ja kategoriat"
+
+msgid "Change customer to person contact mode"
+msgstr "Aseta asiakkuus henkilöasiakas tilaan"
+
+msgid "Change customer to company contact mode"
+msgstr "Aseta asiakkuus yritysasiakas tila"
+
+msgid "Change customer mode to guest"
+msgstr "Aseta asiakkuus vierastilaan"
```

Comment:
The English part: This is basically the same as those two above but few twists to words. I would like to see these unified:
* `Change customer mode to person contact`
* `Change customer mode to company contact`
* `Change customer mode to guest`

Better would be `Show site as xxx` or `Use site as xxx` for example `Use site as person contact` etc.